### ISSUE TITLE
Discovery - Can migration of /generatePDF remove the blocker to ship Toxic Exposure?

### ISSUE THREAD

**Issue by RakshindaAslam** [department-of-veterans-affairs/va.gov-team#77647]
## Issue Description
Discover whether /generatePDF endpoint migration can allow submission of Toxic Exposure section without the migration of /submit endpoint.
 
---
## Tasks
- [ ] _What work is necessary for this story to be completed?_

## Acceptance Criteria
- [ ] Determine if /generatePDF endpoint migration will remove /submit migration blocker for shipping Toxic Exposure section for 526
---
## How to configure this issue
- [ ] **Attached to a Milestone** (when will this be completed?)
- [ ] **Attached to an Epic** (what body of work is this a part of?)
- [ ] **Labeled with Team** (`product support`, `analytics-insights`, `operations`, `service-design`, `Console-Services`, `tools-fe`)
- [ ] **Labeled with Practice Area** (`backend`, `frontend`, `devops`, `design`, `research`, `product`, `ia`, `qa`, `analytics`, `contact center`, `research`, `accessibility`, `content`)
- [ ] **Labeled with Type** (`bug`, `request`, `discovery`, `documentation`, etc.)

**Comment by aurora-a-k-a-lightning** [comment]
## Acceptance Criteria Answers
- [x] _Determine if /generatePDF endpoint migration will remove /submit migration blocker for shipping Toxic Exposure section for 526_
TL;DR: Yes.
Using the generatePDF endpoint would unblock shipping Toxic Exposure.

- [x] _Determine different options to deploy TE without submit with T shirt size  effort and assumptions_
Both options are _**huge**_ T-Shirt sized efforts with different considerations and "gotchas"

- [x] Discuss and document feasibility of each option: 
## Option 1
### EVSS Submitted Claim with no PDF generation
1. set "autoCestPDFGenerationDisabled" to `true` in the EVSS submit request body
2. Send to EVSS endpoint
3. get back claim_id (submitted_claim_id) and save to database
4. transform data using 526 EVSS -> Lighthouse data transformer
5. send Lighthouse formatted request to /generatePDF endpoint
6. receive PDF; save to temp path
7. with claim_id (submitted_claim_id) in the request body, send received 526 PDF to LH benefits-documents/documents endpoint
8. get back 200 response
9. poll for upload success
10. trigger ancillary jobs (as usual)

## Option 2 (Not recommended)
 ### Solely use LH benefits intake
A few notes:
1. Please keep in mind this is an _**Option**_ to remove the Toxic Exposure blocker
2. This is not our primary choice as it places the form 526 PDF in Central Mail
3. This would remove the need for a "Primary" and "Backup" path - this would just be how form 526s get submitted.

1. Remove call to EVSS to get claim_id (submitted_claim_id)
2. transform data using 526 EVSS -> Lighthouse data transformer
3. send Lighthouse formatted request to /generatePDF endpoint
4. receive PDF; save to temp path
5. Send PDF to Central Mail via CentralMail Service or LH benefits-intake api (/uploads then /path)

- [x] Diagram of how the claim could get submitted with the LH pdf and without the LH /submit endpoint
<img width="717" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBM1gvQXc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--c2b979b0f16432dc75658a61a4fa3acc1337b4e7/DBex%20Diagrams%20-%20%2376728%20-%20Submission%20Solutions.jpeg" alt="DBex Diagrams - #76728 - Submission Solutions.jpeg" />

- [x] Would this also work with the backup process?
Yes. 
The backup process can also use the LH /generatePdf endpoint.
It would look almost the same as Option 2

1. transform data using 526 EVSS -> Lighthouse data transformer
2. send Lighthouse formatted request to /generatePDF endpoint
3. receive PDF; save to temp path
4. Send PDF to Central Mail via CentralMail Service or LH benefits-intake api (/uploads then /path)
